flutter_secure_storage/pubspec.yaml

```diff
@@ -35,7 +35,7 @@ dependencies:
   flutter_secure_storage_linux: ^1.0.0
   flutter_secure_storage_macos: ^1.0.0
   flutter_secure_storage_platform_interface: ^1.0.0
-  flutter_secure_storage_web: ^1.0.1
+  flutter_secure_storage_web: ^1.0.0
```

Comment:
Why did you downgrade the web version?

Reply:
Sorry, I didn't explain this in the original post. I was having an issue with version solving, but today I reverted the change and everything works fine. My apologies.